packages/camera/camera/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.8.2
+
+* Complete rewrite of Android plugin to fix all capture, focus, flash, and exposure issues.
```

Comment:
s/all/many/

Reply:
Removed from this PR and fixed in #4153.